QuoteGeneration/Makefile

```diff
@@ -63,7 +63,7 @@ dcap: $(CHECK_OPT) pce_wrapper quote_wrapper qcnl_wrapper qpl_wrapper qve_wrappe
 opt_check_failed:
 	@echo "ERROR: Please decompress prebuilt enclave package before compiling."
 	@echo "Exiting......"
-	@exit -2
+	@exit 2
```

Comment:
Since this PR is to correct exit code, suggest to use another exit code rather than 2 as 2 is reserved for shell builtins. See reserved exit code list in http://www.tldp.org/LDP/abs/html/exitcodes.html

Reply:
Good idea. Changed to 3, the first non-reserved value. Is it ok now?